bin/runInSdTest.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+VERSION=$1
+
+if [ -z "${VERSION}" ]; then
+    echo "You must give a version to execute command on, for example 7.1 for PHP 7.1 container."
```

Comment:
That not correct.

`echo "You must give a version to execute command on, for example 71 for PHP 7.1 container."`

(Notice the missing dot in version.)